samples_project/Assets/SampleViewer/Scripts/SampleSwitcher.cs

```diff
@@ -118,6 +118,16 @@ private void OnEnable()
     private void PopulateSampleSceneList()
     {
         SceneDropdown.options.Clear();
+
+#if !(USE_OPENXR_PACKAGE)
+        if (SceneList.Contains("VR-Sample")){ 
```

Comment:
Can we use equals instead of contains?

Reply:
Wouldn't you have to use a loop to use equals, it's a List so it's just looking if the List contains a string named VR-Sample, saves having to manually search through the list.